packages/perseus/src/widgets/__tests__/numeric-input.test.ts

```diff
@@ -550,24 +547,23 @@ describe("unionAnswerForms utility function", () => {
         const forms = [
             [
                 {
-                    simplify: "required",
+                    simplify: "required" as const,
                     name: "integer",
-                },
+                } as const,
             ],
             [
                 {
-                    simplify: "required",
+                    simplify: "required" as const,
                     name: "integer",
-                },
+                } as const,
```

Comment:
Not sure if there's a cleaner/more concise way to do this but at least we get rid of a `@ts-expect-error`